common/app/model/content/Atoms.scala

```diff
@@ -86,7 +87,14 @@ object Atoms extends common.Logging {
     content.atoms.map { atoms =>
       val quizzes = extract(atoms.quizzes, atom => { Quiz.make(content.id, atom) })
 
-      val media = extract(atoms.media, atom => { MediaAtom.make(atom) })
+      val media = extract(atoms.media, atom => {
+        val endSlatePath = EndSlateComponents(
+          sectionId = content.sectionId.getOrElse(""),
+          shortUrl = content.fields.flatMap(_.shortUrl).getOrElse(""),
```

Comment:
I think this would be a new bit of work to serve an endslate on embeds as we don't know what makes sense to show. Maybe most popular or latest? But we don't really have any context to do much more than that.